Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+#!groovy
+
+def nodeLabel = 'docker'
+
+/* Only keep the 10 most recent builds. */
+properties([[$class: 'BuildDiscarderProperty',
+                strategy: [$class: 'LogRotator', numToKeepStr: '10']]])
+
+node(nodeLabel) {
```

Comment:
Since this `Jenkinsfile` is no longer using `docker` in any form, this can just be:

```groovy
node {
  /* ... */
}
```